spec/requests/users_controller_spec.rb

```diff
@@ -5873,33 +5931,54 @@ def revoke(user, skip_remote: false)
         get "/u/#{user.username}/user-menu-private-messages"
         expect(response.status).to eq(200)
 
-        notifications = response.parsed_body["notifications"]
-        expect(notifications.map { |notification| notification["id"] }).to contain_exactly(
-          unread_notification.id
-        )
+        unread_notifications = response.parsed_body["unread_notifications"]
+        expect(unread_notifications.map { |notification| notification["id"] }).to eq([
+          unread_pm_notification.id,
+          unread_group_message_summary_notification.id
+        ])
+      end
+
+      it "sends an array of read group_message_summary notifications" do
+        get "/u/#{user.username}/user-menu-private-messages"
+        expect(response.status).to eq(200)
+
+        unread_notifications = response.parsed_body["read_notifications"]
+        expect(unread_notifications.map { |notification| notification["id"] }).to eq([
+          read_group_message_summary_notification.id
```

Comment:
I would recommend asserting against more records to check that the ordering is correct.